src/schemas/models/history.js

```diff
@@ -0,0 +1,55 @@
+const _idSchema = {
+  bsonType: 'objectId',
+  type: 'string',
+  minLength: 24,
+  maxLength: 24,
+  description: 'ID of history entry'
+}
+
+const watchIDSchema = {
+  bsonType: 'objectId',
+  type: 'string',
+  minLength: 24,
+  maxLength: 24,
+  description: 'ID of watch'
+}
+
+const targetIDSchema = {
+  bsonType: 'objectId',
+  type: 'string',
+  minLength: 24,
+  maxLength: 24,
+  description: 'ID of target'
+}
+
+const dataSchema = {
+  bsonType: 'string',
+  type: 'string',
+  description: 'Value'
+}
+
+const createdAtSchema = {
+  bsonType: 'date',
+  type: 'string',
+  format: 'date-time',
+  description: 'Time at which the history was created'
```

Comment:
```suggestion
  description: 'Time at which the history entry was created'
```
Vc t bảo ông check lại xem còn chỗ nào dùng sai thuật ngữ không cơ mà. History là cả lịch sử, entry là 1 bản ghi thôi.